openpgp/packet/packet.go

```diff
@@ -380,7 +380,7 @@ func Read(r io.Reader) (p Packet, err error) {
 		err = errors.UnsupportedError("Symmetrically encrypted packets without MDC are not supported")
 	case packetTypeLiteralData:
 		p = new(LiteralData)
-	case packetTypeUserId:
+	case packetTypeUserID:
 		p = new(UserId)
```

Comment:
Similarly here (UserId / UserID). We might just want to keep `Id` everywhere, linter be damned